laserstudio/laserstudio.py

```diff
@@ -185,11 +186,11 @@ def __init__(self, config: Optional[dict]):
         if window_state is not None:
             self.restoreState(window_state)
 
-    def closeEvent(self, event):
+    def closeEvent(self, a0):
```

Comment:
search/replace mistake?

Reply:
No, for pyQt6, QtWidgets.pyi contains "def closeEvent(self, a0: typing.Optional[QtGui.QCloseEvent]) -> None: ...", so PyLance was complaining that the redefinition of the function does not have the same parameters name.